llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1063,7 +1138,7 @@ void VPlanTransforms::optimizeForVFAndUF(VPlan &Plan, ElementCount BestVF,
 
     VPBlockUtils::connectBlocks(Preheader, Header);
     VPBlockUtils::connectBlocks(ExitingVPBB, Exit);
-    simplifyRecipes(Plan, *CanIVTy);
+    VPlanTransforms::simplifyRecipes(Plan, *CanIVTy);
```

Comment:
unrelated?

Reply:
This is actually necessary - because this logic has been moved into a (static) non-member function we now need the class scope when calling the `simplifyRecipes()`